sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/CertificateClientLiveTests.cs

```diff
@@ -1123,5 +1123,61 @@ private static bool IsExpectedP256KException(Exception ex, CertificateKeyCurveNa
             CertificateTransparency = false,
             ContentType = CertificateContentType.Pkcs12
         };
+
+        [RecordedTest]
+        public async Task StartCreateCertificateWithPreserveCertOrder()
+        {
+            string certName = Recording.GenerateId();
+            CertificateClient client = GetClient();
+
+            var policy = CertificatePolicy.Default;
+            var tags = new Dictionary<string, string>
+            {
+                { "tag1", "value1" },
+                { "tag2", "value2" }
+            };
+
+            CertificateOperation operation = await client.StartCreateCertificateAsync(
+                certName,
+                policy,
+                enabled: true,
+                tags: tags,
+                preserveCertOrder: true);
+
+            Assert.That(operation, Is.Not.Null);
+            Assert.That(operation.Properties.Name, Is.EqualTo(certName));
+
+            KeyVaultCertificateWithPolicy cert = await operation.WaitForCompletionAsync();
+
+            Assert.That(cert, Is.Not.Null);
+            Assert.That(cert.Name, Is.EqualTo(certName));
+            Assert.That(cert.Properties.Enabled, Is.True);
+            Assert.That(cert.Properties.Tags, Is.EquivalentTo(tags));
+        }
+
+        [RecordedTest]
+        public async Task ImportCertificateWithPreserveCertOrder()
```

Comment:
Do we need a test that validates that perserve order is false when set to false? One recommendation is to change this test to take a parameter `([Values(true, false)] bool preserveOrder)` and assert that `cert.PreserveCertOrder` matches that value.